index.html

```diff
@@ -71,96 +71,6 @@ <h3 class="abt_hero">I’am a Software Developer</h3>
     </ul>
   </section>
   <section id="work-section" class="portfolio">
-    <div class="work-title">
-      <h2>My Resent work</h2>
-      <div></div>
-    </div>
-    <div class="container">
-      <div class="work1">
-        <div class="background"></div>
-        <div class="work1-text">
-          <h3 class="work-subtitle">Multi-Post Stories</h3>
-          <p class="works-desk">A daily selection of privately personalized reads; no accounts or sign-ups required. has
-            been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
-            text.</p>
-          <ul class="lang-list">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-          </ul>
```

Comment:
- [ ] Kindly adjust the padding of some of the buttons in this section to match the original design

![image](https://user-images.githubusercontent.com/98453979/216052423-3cf6ab01-9b0e-46a1-9a2d-3779eb842230.png)

### Checkout figma design

![image](https://user-images.githubusercontent.com/98453979/216052536-6ba3dd09-7175-4cbe-8ba7-7b3bcbea242a.png)